.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+<!--
+
+Hello and thanks for contributing!
+
+Before continuing, can you check if the answer is available within:
+
+* The documentation - https://www.mapbox.com/mapbox-navigation-ios/navigation/
+* The example app - https://github.com/mapbox/navigation-ios-examples
+
+-->
+
+<!-- Please note which version of the Mapbox Navigation SDK you are using. -->
+
+**Mapbox Navigation SDK version:**
```

Comment:
Maybe link to [this documentation on how to file a good navigation SDK bug](https://github.com/mapbox/mapbox-navigation-ios/blob/master/CONTRIBUTING.md#reporting-an-issue), since there are multiple places to report an issue.